src/parsers/cdm_parser.jl

```diff
@@ -736,6 +699,71 @@ function services_csv_parser(rsv_raw::DataFrames.DataFrame,gen_raw::DataFrames.D
     return services_dict
 end
 
+"""Return a dictionary that includes a vector of parsed forecast dictionaries."""
+function forecast_csv_parser(ps_dict::Dict, data::Dict)
+    forecasts = Dict(
+        "forecasts" => Vector{Dict}(),
+        "loads" => Vector{Dict}(),
+    )
+
+    # A system can only contain forecasts with one resolution. Skip any that don't
```

Comment:
I need some help here.

Reply:
At a Dict level, we can load forecasts with more than one resolution. We should have some sort of interface that tells the user that more than one resolution is available and choose one. 

We would actually use the resolution of the selected forecast to actually build and test the whole system data.

Reply:
SystemForecasts is currently an immutable struct with a field called resolution.  We could make it mutable, add a field called available_resolutions, and let the user change it.  All forecasts could be stored within the data object.

Reply:
When we create the system we want a single resolution. We don't want to carry around all possible time series for a system.

Reply:
Then we are missing a place to insert a new interface.  The user could specify the resolution to use when calling the System constructor.  But what if they don't know that multiple are available?  Are you suggesting a new, intermediate layer?  We could initially store all time series but force a decision if there are multiple, and then delete the unused ones.  (I don't really like that option.)

Reply:
The parsing code now stores all forecasts regardless of resolution.  The System construction will fail if there are multiple resolutions.  It will be up to the user to choose one resolution and filter out the rest before calling the System constructor.